client/signup/steps/store-features/index.tsx

```diff
@@ -28,6 +29,25 @@ export default function StoreFeaturesStep( props: Props ): React.ReactNode {
 	const subHeaderText = translate( 'Let’s create a website that suits your needs.' );
 	const siteSlug = props.signupDependencies.siteSlug;
 
+	const sitePlanSlug = useSelector( ( state ) => {
+		const site = getSite( state, siteSlug );
+		return site && site.plan && site.plan.product_slug;
```

Comment:
I think you can use optional chaining here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
Awesome! I'd forgotten about this one!